test/karma/test-app/dom-reattach-clone/cmp-deep-slot.tsx

```diff
@@ -2,6 +2,7 @@ import { Component, h } from '@stencil/core';
 
 @Component({
   tag: 'dom-reattach-clone-deep-slot',
+  scoped: true,
```

Comment:
For my own understanding, why do we need to add `scoped` to the components in this test suite?

Reply:
The test uses the `cloneNode` method on a Stencil component, but with the changes to the bootstrap methods in this PR, we only patch native methods for `scoped` components when `experimentalSlotFixes` is enabled.